BUILD.bazel

```diff
@@ -1,14 +1,11 @@
-licenses(["notice"])  # Apache v2
-
 load("@//:bazel/lex.bzl", "genlex")
 load("@//:bazel/bison.bzl", "genyacc")
 
 package(
     default_visibility = ["//visibility:public"],
+    license = "notice",  # Apache v2
 )
 
-PACKAGE_NAME = "com_github_p4lang_p4c"
-
 P4C_BUILD_DEFAULT_COPTS = [
     "-DCONFIG_PKGDATADIR=\\\"external/p4lang_p4c\\\"",
     # This is a bit of a hack, but will work if the binary is executed by Bazel
```

Comment:
I don't really understand this, is this hack necessary? (I know this is not something you changed, just curious).

Reply:
I'd like to understand that as well. Not sure yet.